runtime/src/main/java/io/quarkiverse/cxf/CXFQuarkusServlet.java

```diff
@@ -57,7 +57,7 @@ private Object getInstance(String className) {
         Class<?> classObj = loadClass(className);
         try {
             return classObj.getConstructor().newInstance();
-        } catch (Exception e) {
+        } catch (ReflectiveOperationException | RuntimeException e) {
```

Comment:
The original code was catching `Exception` which means it was implicitly catching `RuntimeException`s too. For that reason I kept `RuntimeException` in the revised catch clause. This happens in a few other places too.
Ideally, catch clauses with `RuntimeException`s should be removed except where strictly necessary, but I didn't want to make that call at this point.